rpcs3/util/sysinfo.cpp

```diff
@@ -177,6 +177,71 @@ bool utils::has_avx512_vnni()
 #endif
 }
 
+bool utils::has_avx10()
+#if defined(ARCH_X64)
```

Comment:
All of the new ifdefs you added need to be moved one line down to be underneath the starting `{` or else this will not compile on ARM

Reply:
Oops, fixed